src/main/java/com/chargerrobotics/subsystems/LimelightSubsystem.java

```diff
@@ -70,21 +93,145 @@ public double getV() {
 		return v;
 	}
 
-	// distance in inches
-	public double distance() {
-		if (v == 0) {
-			return -1.0;
+	public double getS() {
+		return s;
+	}
+
+	public int pl;
```

Comment:
Rather than setting this variable here, since `findPL()` returns the value, create a local variable within `findPL()`.

For example:
```java
public int findPL() {
    int pl = -1;
    ...
    return pl;
```
Then elsewhere:
```java
int pl = findPL();
```

Same goes for many of these other variables that are only used within a particular method.